src/backend/yaas.js

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 
-export async function getDocument(docID) {
+var SERVER_URL = ""
+if (process.env.NODE_ENV == "development"){
+    SERVER_URL = "http://localhost:5000/";
+}else{
+    SERVER_URL = "https://yaas.azurewebsites.net/";
+}
 
-    const response = await fetch("http://localhost:5000/query?docID="+docID);
+export async function getDocument(docID) {
 
+    const response = await fetch(SERVER_URL + "query?docID="+docID);
```

Comment:
Let's add timeouts to these functions. We can define a global timeout variable before line 3 (I think it shouldn't take more than 5 seconds to respond, but we can put 8 for now, just to be conservative)